src/citrine/informatics/workflows/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .design_workflow import *
+from .predictor_evaluation_workflow import *
+from .performance_workflow import *
+from .workflow import *
```

Comment:
I deleted `workflows.py` and moved existing workflows to their own package. Importing the classes here will persevere the existing API and allow us to add new workflows in new files over time instead of accruing code into a single monolithic file (e.g. `predictors.py`).

Reply:
Sounds like we should do the same with predictors. I'll open an issue